src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Managing AWS PrivateLink for a Dedicated Cluster
+summary: Instructions for establishing AWS PrivateLink network access for a cluster.
+toc: true
+docs_area: manage
+---
+
+Amazon Web Services (AWS) PrivateLink allows customers to establish SQL access to their {{ site.data.products.dedicated }} clusters entirely through private AWS infrastructure, without exposure to the public internet, affording enhanced security and performance.
+
+For broader context, refer to [Network Authorization for CockroachDB Cloud Clusters](network-authorization.html)
+
+This page describes the steps to setting up an AWS PrivateLink connection for your {{ site.data.products.dedicated }} cluster from the AWS side. You must also configure the AWS PrivateLink connection from the cluster side. Refer to [Establish VPC Peering or AWS PrivateLink](connect-to-your-cluster.html#establish-vpc-peering-or-aws-privatelink)
+
+{{site.data.alerts.callout_info}}
+If you have multiple clusters, you will have to repeat these steps for each cluster that you want to connect to using AWS PrivateLink.
+{{site.data.alerts.end}}
+
+1.  [Set up a cluster](#set-up-a-cluster)
```

Comment:
These steps here are represented on this page and encompass: "This page describes the steps to setting up an AWS PrivateLink connection for your CockroachDB Dedicated cluster from the AWS side."?

Suggestion: The numbered list goes immediately after this first sentence, so something like:

> This page describes the steps to setting up an AWS PrivateLink connection for your CockroachDB Dedicated cluster from the AWS side. 
> 
> 1. [Set up a cluster](#set-up-a-cluster)
> 1.  [Create an AWS endpoint](#create-an-aws-endpoint)
> 1.  [Verify the endpoint ID](#verify-the-endpoint-id)
> 1.  [Enable private DNS](#enable-private-dns)
> 
> You must also configure the AWS PrivateLink connection from the cluster side. Refer to [Establish VPC Peering or AWS PrivateLink]....
> 
> {{site.data.alerts.callout_info}}
> If you have multiple clusters, you will have to repeat these steps for each cluster that you want to connect to using AWS PrivateLink.
> {{site.data.alerts.end}}